README.md

```diff
@@ -57,4 +57,23 @@ If you are on call or asked to fix an immediate issue, check out our doc on
 [how support works](/playbooks/support.md#readme) and reference our
 [support wiki](https://github.com/artsy/potential/wiki) 🔒 for up-to-date playbooks on how to solve issues.
 
-<a rel="license" href="https://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" />
+<a rel="license" href="https://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>
+
+## About Artsy
+
+[![artsy][footer_logo]][footer_website]
+
+This project is supported by [Artsy][footer_website], a marketplace for buying
+and selling art online. One of our most beloved Engineering principles is being
```

Comment:
This sentence could be restructured to be a little easier to read. Right now, it's "independent clause – imperative call-to-action", and the dash doesn't feel like enough of a separator to me. How about something like:

> Because one of our most beloved Engineering Principles is being Open Source by Default, we share a lot of how we work with the open source community. You can learn more about ...

Or something similar.

I think "Engineering Principles" could also link to the general principles doc – you link to a specific one later, but we can do both 👍

Reply:
Thanks - updated with 718c608! 😎